test/testuninitvar.cpp

```diff
@@ -7294,7 +7294,7 @@ class TestUninitVar : public TestFixture {
                         "    A a;\n"
                         "    x.push_back(a);\n"
                         "}\n");
-        ASSERT_EQUALS("[test.cpp:9]: (error) Uninitialized variable: a.j\n", errout.str());
+        ASSERT_EQUALS("[test.cpp:9]: (error) Uninitialized variable: a\n", errout.str());
```

Comment:
I think that ideally it's better to write a member that is not initialized. It's something explicit that the user can investigate carefully.

Does this PR mean the analysis will be better? Less false negatives or false positives?

Reply:
> I think that ideally it's better to write a member that is not initialized. It's something explicit that the user can investigate carefully.

We write the variables when it is partially initialized, but if all members are  uninitialized we just write the struct. 

> Does this PR mean the analysis will be better?

Yes because we will track aliases better across all of our analyzers not just `SubExpressionAnalyzer`. This should lead to less FPs. And will help enable more FNs in the future as well.